### PR TITLE
Hooks: list hooks supported by docker

### LINKED ISSUE #432: Docker: hooks are not available except for CC_RUN_SUCCEEDED_HOOK

### PR DESCRIPTION
**Pull request by BlackYoup**
Fixes #432

### CODE REVIEW

File: commons/clever-cloud-overview/hooks.md [hunk 1]
```diff
@@ -42,6 +42,18 @@ possible with your build tool, but sometimes it's not possible. Another reason
 to use hooks is to integrate Clever Cloud-specific tasks which don't belong in
 the project's build configuration.
 
+## Exceptions
+
+Some instance type do not have hooks support or only some of them.
```
Comment (urcadox) [inline commons/clever-cloud-overview/hooks.md]: instance types* (do we actually use "instance types" in the documentation elsewhere?)

File: commons/clever-cloud-overview/hooks.md [hunk 2]
```diff
@@ -42,6 +42,18 @@ possible with your build tool, but sometimes it's not possible. Another reason
...
+
+### Docker
+
+For Docker, because you can do pretty much anything yourself inside your Dockerfile, only the following hooks are available:
+
+- CC_RUN_SUCCEEDED_HOOK
+
+If you need support for other hooks, please reach out to our support and explain your use case.
```
Comment (urcadox) [inline commons/clever-cloud-overview/hooks.md]: Do we actually have (an idea of) a way to support more hooks in Docker?

Comment (BlackYoup) [reply commons/clever-cloud-overview/hooks.md]: It depends of the hook but I guess exactly like we do for the `CC_RUN_SUCCEEDED_HOOK`.

File: commons/clever-cloud-overview/hooks.md (hunk 1 above)
Comment (BlackYoup) [reply commons/clever-cloud-overview/hooks.md]: Yeah I wasn't sure about this one either. We use it in https://github.com/CleverCloud/doc.clever-cloud.com/blob/master/commons/clever-cloud-overview/common-application-configuration.md but that's pretty much it. Do you have a better naming idea to describe this?

Comment (urcadox) [reply commons/clever-cloud-overview/hooks.md]: I guess "application type(s)" would be better, at least.

File: commons/clever-cloud-overview/hooks.md (hunk 2 above)
Comment (urcadox) [reply commons/clever-cloud-overview/hooks.md]: Oh right, I forgot it ran inside the container. Fair enough.

File: commons/clever-cloud-overview/hooks.md (hunk 1 above)
Comment (BlackYoup) [reply commons/clever-cloud-overview/hooks.md]: updated

**Pull request merged**